config/samples/secure-db/mysql-tls/kustomization.yaml

```
@@ -5,7 +5,41 @@ resources:
+  - target:
+      group: modelregistry.opendatahub.io
+      version: v1alpha1
+      kind: ModelRegistry
+      name: modelregistry-sample
+    patch: |-
+      - op: add
+        path: /spec/mysql/sslRootCertificateConfigMap
+        value:
+          name: model-registry-db-credential
+          key: ca.crt
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)